app/modules/notifications/schemas.py

```diff
@@ -37,7 +36,7 @@ class Meta:
             Notification.guid.key,
             Notification.is_read.key,
             'sender_name',
-            'sender_email',
+            Notification.sender_guid.key,
             Notification.message_type.key,
```

Comment:
Why do we no longer want email? Seems it might be useful to provide to a recipient so they can begin a conversation about a collaboration request or something.

Reply:
The idea was that this only provides information that is necessary to display the notification without additional requests to Houston. If the user wants to initiate anything with the other user, that is what the sender_guid would be used for. Tagged you in the slack convo when Ben & I agreed this.